starter_code/elevator.js

```diff
@@ -1,19 +1,107 @@
+const Person = require('./person.js');
 class Elevator {
   constructor(){
-    this.floor      = 0;
-    this.MAXFLOOR   = 10;
-    this.requests   = [];
-  }
-
-  start() { }
-  stop() { }
-  update() { }
-  _passengersEnter() { }
-  _passengersLeave() { }
-  floorUp() { }
-  floorDown() { }
-  call() { }
-  log() { }
+    this.floor= 0;
+    this.MAXFLOOR= 10;
+    this.requests= [];
+    this.direction = "";
+    this.passengers =[];
+    this.waitingList = [];
+  }
+
+  start() { 
+    var upd = setInterval(this.update,3000);
```

Comment:
en vez de variables globales que estás declarando con var, deberían ser propiedades de la propia clase Elevator. this.interval por ejemplo